src/NBitcoin/Script.cs

```diff
@@ -1088,13 +1085,11 @@ public enum BitcoinConsensusError
 
         public static bool VerifyScriptConsensus(Script scriptPubKey, Transaction tx, uint nIn, ScriptVerify flags)
         {
-            var err = BitcoinConsensusError.ERR_OK;
-            return VerifyScriptConsensus(scriptPubKey, tx, nIn, flags, out err);
+            return VerifyScriptConsensus(scriptPubKey, tx, nIn, flags, out BitcoinConsensusError unused);
```

Comment:
mmh this one is interesting.
`err` is explicitely set, but it's not used and instead set to the same value in `VerifyScriptConsensus`.
I think it's ok to remove it, as you did.